app/controllers/api/functions.php

```diff
@@ -474,6 +475,38 @@
         ])));
 
         if ($next) {
+            $schedule = Authorization::skip(function () use ($dbForConsole, $project, $function) {
+                return $dbForConsole->findOne('schedules', [
+                    Query::equal('region', [App::getEnv('_APP_REGION')]), // Todo replace with projects region
```

Comment:
We should use the project's region not the current region.

Reply:
The project region is not yet implemented, how you recommend to proceed
with this issue.